app/src/main/java/dev/andrylat/app/App.java

```diff
@@ -0,0 +1,155 @@
+package dev.andrylat.app;
+
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.stream.IntStream;
+
+public class App {
+    private StringBuilder errors;
+    private Scanner scanner;
+    private String brand = "";
+    private static final int validCreditCardNumberLength = 16;
```

Comment:
Please, follow the naming convention: https://google.github.io/styleguide/javaguide.html 

for example, here should be https://google.github.io/styleguide/javaguide.html